docs/guides/migrations.md

```diff
@@ -0,0 +1,83 @@
+<!--[meta]
+section: guides
+title: Migrations
+[meta]-->
+
+## Migrations in Keystone
+
+Migrations are required when you update a List in Keystone after the database has been initialised. **Keystone can't automatically manage these migrations for you** because it doesn't know what to do with the existing data. This guide will explain when and why you need database migrations and offer some basic solutions to get you started. As your project grows to include production deployments, how you manage the seeding and migration of data, will depend on your specific situation. Keystone tries not to limit type of applications you build and therefor cannot offer a "one size fits all" solution to migrations.
```

Comment:
I think the example below explains what you're saying better.  Maybe keep this paragraph as short as possible.  Something like this could be enough:

```suggestion
As your app grows new features, you'll inevitably need to change some of your existing List definitions. When you do, you'll also need to migrate the data and schema in your database backend. 
```